extensions/ql-vscode/test/pure-tests/query-history-info.test.ts

```diff
@@ -144,4 +150,18 @@ describe('Query history info', () => {
       });
     });
   });
+
+  describe('getActionsWorkflowRunUrl', () => {
+    it('should get the run url for remote query history items', () => {
+      const actionsWorkflowRunUrl = getActionsWorkflowRunUrl(remoteQueryHistoryItem);
+
+      expect(actionsWorkflowRunUrl).to.equal(`https://github.com/${remoteQueryHistoryItem.remoteQuery.controllerRepository.owner}/${remoteQueryHistoryItem.remoteQuery.controllerRepository.name}/actions/runs/${remoteQueryHistoryItem.remoteQuery.actionsWorkflowRunId}`);
```

Comment:
(optional) This is a very long line, it'd be nice to break things up a bit e.g. extract some of the item used in the URL into variables.